pkg/operator/daemonset.go

```diff
@@ -143,6 +143,17 @@ func MakeDaemonSet(fb fluentbitv1alpha2.FluentBit, logPath string) *appsv1.Daemo
 		})
 	}
 
+	// Mount Wasm Filter
+	if fb.Spec.WasmFile != (corev1.VolumeSource{}) {
+		ds.Spec.Template.Spec.Volumes = append(ds.Spec.Template.Spec.Volumes, corev1.Volume{
+			Name:         "wasm",
+			VolumeSource: fb.Spec.WasmFile,
+		})
+		ds.Spec.Template.Spec.Containers[0].VolumeMounts = append(ds.Spec.Template.Spec.Containers[0].VolumeMounts, corev1.VolumeMount{
+			Name:      "wasm",
+			MountPath: "/fluent-bit/wasm",
```

Comment:
Does this mean the `WasmPath` must begin with `/fluent-bit/wasm`?

Reply:
Yes. Because I noticed that's what PositionDB does. Maybe more comments should be added or is there a better way?

Reply:
I recommend using [Volumes and VolumesMounts](https://github.com/fluent/fluent-operator/blob/master/apis/fluentbit/v1alpha2/fluentbit_types.go#L70-L72) to mount wasm files, so you can specify any directory.

Reply:
OK,I have updated it